spec/features/admin/edit_tutorial_spec.rb

```diff
@@ -4,7 +4,8 @@
   let(:tutorial) { create(:tutorial) }
   let(:admin)    { create(:admin) }
 
-  scenario "by adding a video", :js do
+  # scenario "by adding a video", :js, :vcr do
+  scenario "by adding a video",  :vcr do
```

Comment:
I am unable to get `:vcr` to work as a flag if there is already a `:js` flag. Removing the `:js` had no effect on testing, to my knowedge

Reply:
I'm glad you left it there as a comment. Maybe will come into play during the javascript lesson this morning